src/eth/rpc.ts

```diff
@@ -50,7 +51,8 @@ const isEOA = async (address: string) => {
 }
 
 const isOrchestrator = (item) => {
-    return item.id === item.delegate.id;
+    // delegate could be null
+    return item.id === item.delegate?.id;
```

Comment:
`delegate` will be null if the delegator fully unbonded such that it currently does not have a delegate.